usage/namespace/namespace.go

```diff
@@ -33,11 +33,11 @@ func deleteExistingExample() {
 	if apiClients == nil {
 		panic("Failed to load api client")
 	}
-	// Pull existing namespace from cluster
+	// Get existing namespace from cluster
```

Comment:
I thinks changed in [usage/namespace/namespace.go](https://github.com/openshift-kni/eco-goinfra/pull/103/files#diff-58fc6fcda40f9069be2b457da77e1ab6e146c5110a23fd5b3113a6331b735c93) were added by mistake in this PR

Reply:
uff, here as well, sorry,it's not exactly me, for some reason it was changed when I did name refactoring for nodes